apps/admin-gui/src/app/admin/pages/admin-page/admin-consent-hubs/admin-consent-hubs.component.html

```diff
@@ -1,11 +1,15 @@
 <div>
   <h1 class="page-subtitle">{{'ADMIN.CONSENT_HUBS.TITLE' | translate}}</h1>
-
+  <perun-web-apps-alert *ngIf="!globalForceConsents" alert_type="warn">
+    {{'ADMIN.CONSENT_HUBS.GLOBAL_DISABLED' | translate}}
+  </perun-web-apps-alert>
   <perun-web-apps-refresh-button (refresh)="refreshTable()"></perun-web-apps-refresh-button>
   <button
     *ngIf="authResolver.isPerunAdmin()"
     (click)="evaluateConsents()"
-    [disabled]="selection.selected.length === 0"
+    [disabled]="!globalForceConsents || selection.selected.length === 0"
+    [matTooltipDisabled]="globalForceConsents"
+    matTooltip="{{'ADMIN.CONSENT_HUBS.TOOLTIP' | translate}}"
```

Comment:
The tooltip on disabled elements will not show up on some browsers. Our usuall workaround is to wrap the button with span and put the tooltip on that.